python/ray/data/_internal/progress_bar.py

```diff
@@ -119,16 +120,25 @@ def fetch_until_complete(self, refs: List[ObjectRef]) -> List[Any]:
     def set_description(self, name: str) -> None:
         if self._bar and name != self._desc:
             self._desc = name
-            self._bar.set_description(self._desc)
+            formatted_progress = format_num(self._progress)
+            formatted_total = (
+                format_num(self._bar.total) if self._bar.total is not None else "??"
```

Comment:
instead of updating the bar description itself and calling `format_num()` for the updated description, maybe we can pass `bar_format` parameter to `tqdm.tqdm()`.  specifically in this `r_bar` section:

```
r_bar='| {format_num(n_fmt)}/{format_num(total_fmt)} [{elapsed}<{remaining}, ' '{rate_fmt}{postfix}]'
```
something like this. then, i think we wouldn't need to manually update the entire description here and in `_get_state()`

Reply:
I have tried, this is current what bar looks like 
![image](https://github.com/user-attachments/assets/716e8e15-f79a-4a3d-8979-4e347cb742f4)
but if we use bar_format, it will lose some infomation here, so I think the simplest solution is just change the number. 
![image](https://github.com/user-attachments/assets/bd15909d-161e-41c3-8b56-dff534b4e515)